3-PB/Documentazione_esterna/Piano_di_progetto/consuntivo.tex

```diff
@@ -1030,3 +1030,232 @@ \subsubsection{Analisi retrospettiva}
+Avendo ottenuto l'approvazione del prodotto dall'azienda proponente il giorno 26 marzo e avendo iniziato l'iter di candidatura alla revisione della Product Baseline nei primissimi giorni di Aprile 2024, si può considerare soddisfatto l'obiettivo temporale del gruppo, che ha sostenuto il primo colloquio conclusivo il giorno 2024/04/09.\\ \\
+
+\noindent \textbf{Budget preventivato e consuntivato}\\
+Al momento della candidatura, il gruppo aveva stabilito che il budget necessario alla realizzazione del progetto ammontasse a 12985€. Durante il colloquio per la Requirements and Technology Baseline, previe riflessioni e considerazioni sul ritmo erogato fino a quel punto, la stessa cifra era stata confermata, pur con qualche rimodulazione delle ore per ruolo previste.\\
+L'obiettivo di terminare il progetto entro il budget limite è stato ampiamente soddisfatto dal team, che è anche riuscito a risparmiare sulla cifra totale spesa, anche considerando un monte orario minore e la mancata partecipazione alla Customer Acceptance.\\ \\
+
+\noindent \textbf{Impegno orario erogato}\\
+Al momento della candidatura, il gruppo aveva stabilito un impegno di 93 ore ciascuno nella realizzazione del progetto.\\
+L'intervallo stabilito dal regolamento del progetto didattico, che prevedeva un impegno singolo compreso tra le 80 e le 95 ore, è stato rispettato da tutti i membri.\\
+La principale causa del minore numero di ore spese nel progetto, rispetto all'assunzione di impegni iniziale, è da ricercare nella mancata partecipazione alla Customer Acceptance.\\
+Il minore numero di ore erogate non ha però in alcun modo avuto ripercussioni sulla qualità, bontà e completezza del prodotto finale del progetto, che è risultato abbondantemente soddisfacente sia per il gruppo tutto che per l'azienda proponente, la quale ha apprezzato l'impegno preso e rispettato di implementare un elevato quantitativo di requisiti non obbligatori.
+
+\newpage
+\subsubsection{Retrospettiva globale}
+Arrivati alla conclusione di questo impegno e analizzando i risultati ottenuti, in relazione alle aspettative e agli obiettivi prefissati, il progetto appena terminato risulta essere complessivamente molto positivo.\\
+Il gruppo, come precedentemente esposto, è riuscito a rispettare i tempi e i costi stabiliti all'inizio del progetto, così come le ore di impegno erogate da ogni membro rispettano i termini disposti dal regolamento. La solidissima base di partenza per la Product Baseline, creata dal gruppo con le attività svolte nella Requirements and Technology Baseline, ha fatto sì che i dubbi emersi nel colloquio di quest'ultimo riguardo al budget a disposizione per completare il progetto non si traducessero in una reale problematica.\\
```

Comment:
"La solidissima base di partenza per la Product Baseline, creata dal gruppo con le attività svolte nella Requirements and Technology Baseline, ha fatto sì che i dubbi emersi nel colloquio di quest'ultimo riguardo al budget a disposizione..." non ho capito a cosa si riferisce "quest'ultimo", se è l'RTB prima ne parlavi al femminile, quindi quest'ultima